documentation/git-secrets.md

```diff
@@ -0,0 +1,20 @@
+# Git secrets
```

Comment:
I would call this document by what it does more than what tool it uses.

for instance instead of DfE Sign-in we have Authentication
instad of Algolia we have Search

This could be called secrets detection, or something like that